src/main/java/org/broadinstitute/dsm/util/ElasticSearchUtil.java

```diff
@@ -443,7 +443,7 @@ public static void updateRequest(@NonNull String ddpParticipantId, String index,
                     .doc(objectsMapES)
                     .docAsUpsert(true);
 
-            UpdateResponse updateResponse = client.update(updateRequest, RequestOptions.DEFAULT);
+//            UpdateResponse updateResponse = client.update(updateRequest, RequestOptions.DEFAULT);
```

Comment:
is the update not needed?

Reply:
Oh, I had it commented out while testing and forgot to bring it back, I'll change it